src/sql/Cargo.toml

```diff
@@ -9,12 +9,15 @@ publish = false
 [dependencies]
 anyhow = "1.0.52"
 aws-arn = "0.2.0"
+bytes = "1.1.0"
 chrono = { version = "0.4.0", default-features = false, features = ["clock", "std"] }
 csv = "1.1.6"
 dec = "0.4.8"
 enum-kinds = "0.5.1"
 futures = "0.3.21"
 globset = "0.4.8"
+hex = "0.4.3"
+mz-interchange = { path = "../interchange" }
```

Comment:
did you mean to move `mz-interchange` here?

Reply:
I was trying to preserve some semblance of alphabetical sorting, this set of commits straddled the renaming of the mz crates so its possible this is showing up as a green line because of that, I'll take a closer look though.